arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,867 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Visual Studio Resource Provider Client",
+    "version": "2014-04-01-preview",
+    "description": "Use these APIs to manage Visual Studio Team Services resources through the Azure Resource Manager. All task operations conform to the HTTP/1.1 protocol specification and each operation returns an x-ms-request-id header that can be used to obtain information about the request. You must make sure that requests made to these resources are secure. For more information, see https://docs.microsoft.com/en-us/rest/api/index."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.VisualStudio/operations": {
+      "get": {
+        "tags": [
+          "accountResourceOperations"
+        ],
+        "summary": "GetOperationResources",
+        "description": "",
+        "operationId": "getOperationResources_List",
+        "produces": [
+          "application/json"
+        ],
```

Comment:
It is expected to define `ApiVersionParameter` for each path. In the examples, I see it is specified for various API calls. Please include it parameter section of all operations path.

```
        "parameters": [
          {
            "$ref": "#/parameters/ApiVersionParameter"
          },
          // Other params
        ],
```